general_tools/smartquotes.py

```diff
@@ -16,13 +16,14 @@
 from __future__ import unicode_literals
 import shlex
 from subprocess import *
+import sys
 
 
 def smartquotes(text):
     """
     Runs text through pandoc for smartquote correction.
     """
-    command = shlex.split('/bin/pandoc --smart -t plain')
+    command = shlex.split('pandoc --smart -t plain')
```

Comment:
I would think you can run a command without absolute path on Lambda. Never had a problem otherwise.